src/app/fsat/fsat-report/detailed-results/detailed-results.component.html

```diff
@@ -0,0 +1,832 @@
+
+            <tr>
+                <td>
+                    Degree of Saturation
+                </td>
+                <td [ngClass]="{'selected-modification': selectedModificationIndex == -1}">
+                    <span *ngIf="getPsychrometricResults(fsat)">
+                        {{getPsychrometricResults(fsat).saturationDegree | number:'1.0-3' }}
+                    </span>
+                    <span *ngIf="!getPsychrometricResults(fsat)">
+                        &mdash;
+                    </span>
+                </td>
+                <td *ngFor="let modification of fsat.modifications; let index = index;"
+                    [ngClass]="{'selected-modification': index == selectedModificationIndex}">
+                    <span *ngIf="getPsychrometricResults(modification.fsat)">
+                        {{getPsychrometricResults(modification.fsat).saturationDegree | number:'1.0-3' }}
+                    </span>
+                    <span *ngIf="!getPsychrometricResults(modification.fsat)">
+                        &mdash;
+                    </span>
+                </td>
+            </tr>
+
+            <tr>
+                <td>
+                    Humidity Ratio
+                </td>
+                <td [ngClass]="{'selected-modification': selectedModificationIndex == -1}">
+                    <span *ngIf="getPsychrometricResults(fsat)">
+                        {{getPsychrometricResults(fsat).humidityRatio | number:'1.0-4' }}
+                    </span>
+                    <span *ngIf="!getPsychrometricResults(fsat)">
+                        &mdash;
+                    </span>
+                </td>
+                <td *ngFor="let modification of fsat.modifications; let index = index;"
+                    [ngClass]="{'selected-modification': index == selectedModificationIndex}">
+                    <span *ngIf="getPsychrometricResults(modification.fsat)">
+                        {{getPsychrometricResults(modification.fsat).humidityRatio | number:'1.0-4' }}
+                    </span>
+                    <span *ngIf="!getPsychrometricResults(modification.fsat)">
+                        &mdash;
+                    </span>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+
+<div class="d-flex flex-column w-100 align-items-center pt-2 pb-2" *ngIf="fsat.outputs.planeResults">
+    <div class="btn-group">
+        <button type="button" class="btn btn-secondary btn-padding" (click)="setPressureCalcType('static')"
+            [ngClass]="{'active': fanAnalysisService.pressureCalcResultType == 'static'}">Use Static Pressure</button>
+        <button type="button" class="btn btn-secondary btn-padding" (click)="setPressureCalcType('total')"
+            [ngClass]="{'active': fanAnalysisService.pressureCalcResultType == 'total'}">Use Total Pressure</button>
+    </div>
+
+    <label class="group-label">Baseline Assessment Data</label>
+    <table class="table table-hover table-striped">
```

Comment:
Unless Kristina mentioned otherwise, we won't need this table with pressure readings